src/components/projects/view/staff-card.svelte

```diff
@@ -1,25 +1,26 @@
 <script>
-  import { createEventDispatcher } from 'svelte';
-  import Card from 'ui/card.svelte';
+  import { createEventDispatcher, getContext } from 'svelte';
+  import { Card, TextField, Button } from 'attractions';
   import Labeled from 'ui/labeled.svelte';
-  import TextField from 'ui/text-field.svelte';
-  import Button from 'ui/button.svelte';
   import ProjectStages from '@/constants/backend/project-lifetime-stages.js';
   import s from '@/utils/plural-s.js';
 
   export let account;
   export let moderator;
-  export let projectStage;
-  export let application;
-  export let isCreator;
-  export let review = false;
+  export let moderation;
+  export let project;
 
+  const review = getContext('review-mode');
+  const isCreator = moderator.email === $project.creator.email;
```

Comment:
`$project` is an auto-subscription to the `project` store, check out https://svelte.dev/tutorial/auto-subscriptions

Reply:
I know about that, but I thought there had to be a line like `const { project } = stores();` or something similar. Or is it because the `project` prop is already such store?

Reply:
Yes, it takes it from a prop.